packages/wallets/src/features/cashier/modules/TransactionStatus/TransactionStatus.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useCallback, useMemo } from 'react';
+import { useActiveWalletAccount } from '@deriv/api';
+import { WalletText } from '../../../../components/Base';
+import { Loader } from '../../../../components/Loader';
+import Warning from '../../../../public/images/warning.svg';
+import { THooks } from '../../../../types';
+import { TransactionStatusError } from './components/TransactionStatusError';
+import { TransactionStatusSuccess } from './components/TransactionStatusSuccess';
+import useRecentTransactions from './hooks/useRecentTransactions';
+import './TransactionStatus.scss';
+
+type TTransactionStatus = {
+    transactionType?: THooks.CryptoTransactions['transaction_type'];
+};
+
+const TransactionStatus: React.FC<TTransactionStatus> = ({ transactionType }) => {
+    const {
+        error: recentTransactionsError,
+        isLoading: isTransactionsLoading,
+        recentTransactions,
+        refresh: refreshRecentTransactions,
+    } = useRecentTransactions(transactionType);
+    const {
+        data: wallet,
+        error: activeWalletAccountError,
+        isLoading: isActiveWalletAccountLoading,
+        refetch,
+    } = useActiveWalletAccount();
+
+    const isLoading = useMemo(
+        () => isTransactionsLoading || isActiveWalletAccountLoading,
+        [isTransactionsLoading, isActiveWalletAccountLoading]
+    );
+
+    const isError = useMemo(
+        () => !!activeWalletAccountError || !!recentTransactionsError,
+        [activeWalletAccountError, recentTransactionsError]
+    );
+
+    const refresh = useCallback(() => {
+        refreshRecentTransactions();
+        refetch();
+    }, [refetch, refreshRecentTransactions]);
+
+    return (
+        <div className='wallets-transaction-status'>
+            <div className='wallets-transaction-status__header'>
+                <WalletText weight='bold'>Transaction status</WalletText>
+                {isError && <Warning />}
+            </div>
+            <div className='wallets-transaction-status__divider' />
```

Comment:
Base component `Divider` can be used here instead

Reply:
Using `Divider` now